packages/client-python/src/rocketride/_engine/resolver.py

```
@@ -0,0 +1,94 @@
+            # Use tomllib (3.11+) or tomli as fallback
+            try:
+                import tomllib
+            except ImportError:
+                import tomli as tomllib
+
+            with open(pyproject, 'rb') as f:
+                data = tomllib.load(f)
+
+            return data['tool']['rocketride']['engine-compatible']
+        except (KeyError, Exception):
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)